.env.dev

```diff
@@ -1,6 +1,10 @@
 PORT=5000
 NODE_ENV="development"
 JWT_SECRET="thisismysupersecrettokenjustkidding"
-DATABASE_URL="mongodb://localhost:27017/donut-development"
+DATABASE_URL="mongodb+srv://asel123:asel123@socialmediaappcluster-leoir.mongodb.net/example"
```

Comment:
please change this to local, I think you forgot to remove your personal credentials